evaporate.js

```diff
@@ -223,8 +224,22 @@
 
         //con.simulateStalling =  true
 
-        _.add = function (file,  pConfig) {
-            var c = extend(pConfig, {});
+        _.add = function (pFile,  pConfig) {
+            var file,
+                config;
+            
+           if (arguments.length === 1) {
```

Comment:
Good point. I would prefer not to do it but it's clear from feedback that people followed the examples, so if we make a breaking change to the method signature, anyone who upgrades will need to fix their apps first, so I think that the added complexity is the lesser of the two evils.